pkg/app/pipectl/cmd/quickstart/quickstart.go

```diff
@@ -87,36 +100,195 @@ func (c *command) run(ctx context.Context, input cli.Input) error {
+	if err = c.installPiped(ctx, helm, input); err != nil {
+		input.Logger.Error("Failed to install piped!!")
+		return err
+	}
+
+	return nil
+}
+
+func (c *command) installControlPlane(ctx context.Context, helm string, input cli.Input) error {
+	input.Logger.Info("Installing the controlplane in quickstart mode...")
+
+	args := []string{
+		"upgrade",
+		"--install",
+		helmControlPlaneReleaseName,
+		helmChartControlPlaneRepoName,
+		"--version",
+		c.version,
+		"--namespace",
+		c.namespace,
+		"--create-namespace",
+		"--values",
+		fmt.Sprintf(helmQuickstartValueRemotePath, c.version),
+	}
+
+	var stderr, stdout bytes.Buffer
+	cmd := exec.CommandContext(ctx, helm, args...)
+	cmd.Stderr = &stderr
+	cmd.Stdout = &stdout
+
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("%w: %s", err, stderr.String())
+	}
+
+	input.Logger.Info(stdout.String())
+	input.Logger.Info("Intalled the controlplane successfully!")
+
+	return nil
+}
+
+func (c *command) installPiped(ctx context.Context, helm string, input cli.Input) error {
+	input.Logger.Info("\nInstalling the piped for quickstart...")
+
+	input.Logger.Info("\nOpenning PipeCD control plane at http://localhost:8080/\nPlease login using the following account:\n- Username: hello-pipecd\n- Password: hello-pipecd\nFor more information refer to https://pipecd.dev/docs/quickstart/\n")
+
+	if err := openbrowser(controlPlaneLocalhost); err != nil {
+		return fmt.Errorf("failed to open PipeCD control plane: %w", err)
+	}
+
+	input.Logger.Info("Fill up your registered Piped information:")
+
+	pipedID := getPromptInput(pipedIDLabel)
+	pipedKey := getPromptInput(pipedKeyLabel)
+	sourceRepo := getPromptInput(pipedGitRemoteRepo)
+
+	args := []string{
+		"upgrade",
+		"--install",
+		helmPipedReleaseName,
+		helmChartPipedRepoName,
+		"--version",
+		c.version,
+		"--namespace",
+		c.namespace,
+		"--set",
+		"quickstart.enabled=true",
+		"--set",
+		fmt.Sprintf("quickstart.pipedId=%s", pipedID),
+		"--set",
+		fmt.Sprintf("secret.data.piped-key=%s", pipedKey),
+		"--set",
+		fmt.Sprintf("quickstart.gitRepoRemote=%s", sourceRepo),
+	}
+
+	var stderr, stdout bytes.Buffer
+	cmd := exec.CommandContext(ctx, helm, args...)
+	cmd.Stderr = &stderr
+	cmd.Stdout = &stdout
+
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("%w: %s", err, stderr.String())
+	}
+
+	input.Logger.Info(stdout.String())
+	input.Logger.Info("Intalled the piped successfully!")
+
+	return nil
+}
+
+func (c *command) exposeService(ctx context.Context) error {
+	kubectl, err := c.getKubectl()
+	if err != nil {
+		return fmt.Errorf("failed to prepare required tool (kubectl) for installation: %v", err)
+	}
+
+	args := []string{
+		"port-forward",
+		"svc/pipecd",
+		"8080",
+		"-n",
+		c.namespace,
+	}
+	var stderr bytes.Buffer
+	cmd := exec.CommandContext(ctx, kubectl, args...)
+	cmd.Stderr = &stderr
+
+	// Wait the control plane service to be ready.
+	// TODO: Find a better way to handle this instead of time sleep
+	time.Sleep(time.Minute)
```

Comment:
How about using `kubectl rollout status` command or a `--timeout` option for helm?